bool pivot_sliding_left     # True if pivot sliding left
bool pivot_sliding_right    # True if pivot sliding right           
bool pivot_sliding          # pivot_sliding_left || pivot_sliding_right

bool robot_sliding_left     # True if pivot sliding left
bool robot_sliding_right    # True if pivot sliding right           
bool robot_sliding          # robot_sliding_left || robot_sliding_right